easybuild/easyconfigs/r/ReFrame/ReFrame-3.5.3.eb

```diff
@@ -0,0 +1,65 @@
+easyblock = 'PythonBundle'
+
+name = 'ReFrame'
+version = '3.5.3'
+
+homepage = 'https://github.com/eth-cscs/reframe'
+description = '''ReFrame is a framework for writing regression tests for HPC systems.'''
+
+toolchain = SYSTEM
+
+allow_system_deps = [('Python', SYS_PYTHON_VERSION)]
+
+req_py_majver = 3
+req_py_minver = 6
+
+use_pip = True
+
+exts_list = [
+    ('pip', '21.0.1', {
```

Comment:
This is not actually installed properly the last time I tested this, if pip is not available in calling Python the build will fail.

Reply:
I see `software/ReFrame/3.5.3/bin/pip` installed?

Reply:
Yes, but if one of the subsequent modules tries to use it for installation, they fail.

Reply:
You know Python better than me, but the pip gets installed with an eggdir that doesn't get picked up with `use_pip`, so `bin/pip` is there but not the module.

Reply:
See https://github.com/easybuilders/easybuild-easyconfigs/issues/12326

Reply:
The `pip --version` check that is done right before installing the first extension with `pip install` works fine though:

```
== 2021-04-22 11:38:51,955 run.py:571 DEBUG cmd "pip --version" exited with exit code 0 and output:
pip 21.0.1 from /user/gent/400/vsc40023/eb_arcaninescratch/CO7/skylake-ib/software/ReFrame/3.5.3/lib/python3.6/site-packages/pip-21.0.1-py3.6.egg/pip (python 3.6)
```

So I don't think the `.egg` directory doesn't seem to be a problem.

This is with Python 3.6 though, maybe it's only a problem with more recent Python versions?